File: src/main/java/com/loltft/rudefriend/dto/board/BoardListResponse.java
package com.loltft.rudefriend.dto.board;

import java.util.List;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Schema(description = "게시글 목록 응답 DTO")
public class BoardListResponse {

  private Integer total;
  private List<BoardResponse> boards;
}

File: src/main/java/com/loltft/rudefriend/dto/board/BoardListResponse.kt
package com.loltft.rudefriend.dto.board

import io.swagger.v3.oas.annotations.media.Schema


@Schema(description = "게시글 목록 응답 DTO")
data class BoardListResponse(
    var total: Int? = null,
    var boards: MutableList<BoardResponse?>? = null
)

